### PR TITLE
Fix ssl request certificate error

### PR DESCRIPTION
**Pull request by w00lf**
Fallback to download Mozzila cacert.pem and use it as default cert when binary encounters ssl cert error. This is actually a well-known problem for ruby applications, in ruby packer repo there is a lengthy discussion about possible solutions:

https://github.com/pmq20/ruby-packer/issues/10

Also need to update vendor openssl version in ruby-packer and use it

### CODE REVIEW

**Comment by w00lf** [comment]
#9

**Comment by ronaldtse** [comment]
@w00lf thanks for this. However, does this happen on every request? It's a burden to have to download CA root certs on every single request. I'd rather have the root certs bundled in the application?

Expired certificates shouldn't be a problem, OCVP is supposed to handle them?

**Comment by ronaldtse** [comment]
@w00lf perhaps we can merge this in soon?

**Comment by w00lf** [comment]
> @w00lf perhaps we can merge this in soon?

I am working on embedding ssl cert into build, will push changes soon.

**Comment by ronaldtse** [comment]
Sounds good, look forward to that.

**Comment by ronaldtse** [comment]
@w00lf there's a typo: `cacert.pem.mozzila` . Should be "mozilla". Could you fix it? Thanks.

File: bin/metanorma [hunk 1]
```diff
@@ -3,19 +3,56 @@
...
+  sassc_path = Bundler::CLI::Common.select_spec('sassc', :regex_match).full_gem_path
   File
     .new(LIBSASS_TEMP_WRAPPER, 'wb')
     .puts(File.read(File.join(sassc_path, 'ext', LIBSASS_TEMP_WRAPPER)))
 
+  # Check ssl availability, if not use vendor ssl certificate
+  begin
+    Net::HTTP.get(URI('https://www.iso.org/'))
+  rescue OpenSSL::SSL::SSLError
+    puts('Canot use ssl requests, instaling custom certificate') if DEBUG
```
Comment (ronaldtse) [inline bin/metanorma]: "Cannot" instead of "Canot". "SSL" instead of "ssl".

Comment (w00lf) [reply bin/metanorma]: Oh gosh, that's embarrassing. Will fix in v0.0.8

Comment (ronaldtse) [reply bin/metanorma]: No worries, sometimes a review would help before merging 😉  Thanks @w00lf!

**Pull request merged**